bundles/org.openhab.binding.nuki/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -46,7 +46,7 @@
 		<channels>
 			<channel id="lock" typeId="smartlockLock"/>
 			<channel id="lockState" typeId="smartlockState"/>
-			<channel id="lowBattery" typeId="system.low-battery"/>
+			<channel id="lowBattery" typeId="system.low-battery"/><channel typeId="smartlockDoorState" id="doorsensorState"></channel>
```

Comment:
```suggestion
			<channel id="lowBattery" typeId="system.low-battery"/>
			<channel typeId="smartlockDoorState" id="doorsensorState"/>
```

But this might also be fixed when you run `mvn spotless:apply` on the binding